src/SecurityDetails/SecurityDetailsOptions.ts

```diff
@@ -64,13 +68,31 @@ export interface SecurityDetailsMetadata extends MorningstarMetadata {
     date?: string;
 }
 
+export interface SecurityCompareMetadata extends Metadata {
+    ids?: string[];
+    isins?: string[];
+}
+
 
 export interface SecurityDetailsOptions extends MorningstarOptions {
     security?: MorningstarSecurityOptions,
     viewId?: string,
     converter?: SecurityDetailsConverterOptions
 }
 
+
+interface SecurityCompareSecurity extends Omit<MorningstarSecurityOptions, 'id'> {
```

Comment:
Q (everyone):
Not convinced here, depends what the team thinks of it:
Should we follow the hierarchical extending approach, instead of Omits? We have full control over interfaces here. At the same time it is only about removing `id` and adding `ids` 🤔

Reply:
@kamil-musialowski @bm64 @jedrzejruta

Reply:
I don't mind using omit in MS repo, I think it's quite clean.